A fix by Keiichi Oono <keiichi.oono@oracle.com>:
ftp://ftp.vim.org/pub/vim/patches/7.4/7.4.520

---
diff --git src/mbyte.c src/mbyte.c
--- src/mbyte.c
+++ src/mbyte.c
@@ -405,6 +405,7 @@ enc_alias_table[] =
     {"unix-jis",	IDX_EUC_JP},
     {"ujis",		IDX_EUC_JP},
     {"shift-jis",	IDX_SJIS},
+    {"pck",		IDX_SJIS},	/* Sun: PCK */
     {"euckr",		IDX_EUC_KR},
     {"5601",		IDX_EUC_KR},	/* Sun: KS C 5601 */
     {"euccn",		IDX_EUC_CN},
